### DISCUSSION TITLE
Mouse scroll in vi mode

### DISCUSSION THREAD

**Discussion by noahfraiture** [nushell/nushell#14115] [Q&A]
Hello,
I use the vi mode and I would like to still be able to see my terminal scrollback, however, all it does is going through my command history on mouse scroll. Is it possible to disable that ?

**Comment by noahfraiture** [comment]
Ok it seems to be a terminal related, thus the wrong repo, sorry

**Reply by ghost** [reply]
it might also be caused by `tmux`, `abduco`, `screen`, etc if you use such a program
